src/Features/VisualBasic/Portable/Completion/CompletionProviders/TypeImportCompletionProvider.vb

```diff
@@ -0,0 +1,41 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Threading
+Imports Microsoft.CodeAnalysis.Completion.Providers
+Imports Microsoft.CodeAnalysis.Options
+Imports Microsoft.CodeAnalysis.Shared.Extensions.ContextQuery
+Imports Microsoft.CodeAnalysis.Text
+Imports Microsoft.CodeAnalysis.VisualBasic.Extensions.ContextQuery
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+Imports System.Collections.Immutable
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.Providers
+
+    Friend NotInheritable Class TypeImportCompletionProvider
+        Inherits AbstractTypeImportCompletionProvider
+
+        Friend Overrides Function IsInsertionTrigger(text As SourceText, characterPosition As Integer, options As OptionSet) As Boolean
+            Return CompletionUtilities.IsDefaultTriggerCharacterOrParen(text, characterPosition, options)
```

Comment:
interesting.  i'm curious why C# and VB differ here.  but i can see that SymbolCompletion does this.  so i guess it makes sense that this should too.